app/queries/student_details_resolver.rb

```diff
@@ -81,8 +86,12 @@ def submissions
     @submissions ||= student.timeline_events.joins(:target).where(targets: { id: current_course_targets })
   end
 
+  def latest_submission
```

Comment:
The method name needs to be pluralized

Reply:
...and the memoizing instance variable. :)

Reply:
I was about to ask that @harigopal  😄